doc/developer/mzcompose.md

```diff
@@ -114,6 +114,30 @@ from materialize.mzcompose import PythonService
 SERVICES = [PythonService(image = "vendor/container")]
 ```
 
+## Dealing with volumes
+
+All compositions have a set of default volumes, but it is possible to add to these or completely replace them.
```

Comment:
Completely replacing them seems like a footgun given the way that various services implicitly depend upon them in their default configuration. Can we start with just `VOLUMES_EXTRA`?

Reply:
Yeah that's easy, I was just trying to be complete.

Reply:
Yeah it'd be great to find some way to limit the set of volumes added to those that are actually referenced, or something, but that seems like a project for another day.